drizzlepac/haputils/product.py

```diff
@@ -62,6 +62,9 @@ def __init__(self, prop_id, obset_id, instrument, detector, filename, filetype,
         # this attribute is updated in the hapsequncer.py module (run_hla_processing()).
         self.configobj_pars = None
 
+        # Define HAPLEVEL value for this product
+        self.haplevel = 1
+
```

Comment:
Please put some documentation into this module (at top?) which explains what haplevel means, in terms of product and associated numeric value.

Reply:
Addressed in bc2ac06.